### PR TITLE
organization settings

### PR DESCRIPTION
**Pull request by fiquick**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-550.surge.sh

Updated pages:
- [Single Sign-On (SSO)](https://neo4j-docs-aura-550.surge.sh/aura/platform/security/single-sign-on)

File: modules/ROOT/pages/platform/security/single-sign-on.adoc [hunk 1]
```diff
@@ -177,10 +177,24 @@ Accessing Aura with SSO requires:
...
+.Select Organization settings via the cog wheel
+[.shadow]
+image::organizationsettings.png[]
+
+Then, from the organization settings, select *SSO Configuration* to bring up the *New SSO Configuration* dialogue. [Figure 2]. 
+Use the check boxes to "Use as a log in for the Organization" or "Use as login method for instances with projects in this Org".
+You can check both check boxes, for Organization and Instance access here.
+
+Enter basic SSO config information (this can all be retrieved from the IdP).
+Entering the Discovery URI will pre-fill the fields below.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/single-sign-on.adoc:195]:
```suggestion
Entering the Discovery URI pre-fills the fields below. 
```

File: modules/ROOT/pages/platform/security/single-sign-on.adoc [hunk 2]
```diff
@@ -177,10 +177,24 @@ Accessing Aura with SSO requires:
 * Authorization Code Flow
 * A publicly accessible IdP server
 
-To configure SSO, go to *Aura Console > Settings > SSO Configuration.*
-
 To create an SSO Configuration either a Discovery URI or a combination of Issuer, Authorization Endpoint, Token Endpoint and JWKS URI is required.
 
+== Create a new SSO configuration
+
+When you're looking at a project, select the cog wheel from the breadcrumb navigation [Figure 1].
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/single-sign-on.adoc:184]:
```suggestion
From the *Organization settings*, go to *Single Sign-On* and use the *SSO Configuration* button to set up a new SSO configuration.
```

File: modules/ROOT/pages/platform/security/single-sign-on.adoc [hunk 3]
```diff
@@ -177,10 +177,24 @@ Accessing Aura with SSO requires:
...
+
+.Select Organization settings via the cog wheel
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/single-sign-on.adoc:186]:
```suggestion
.Organization settings
```

File: modules/ROOT/pages/platform/security/single-sign-on.adoc [hunk 4]
```diff
@@ -177,10 +177,24 @@ Accessing Aura with SSO requires:
...
+[.shadow]
+image::organizationsettings.png[]
+
+Then, from the organization settings, select *SSO Configuration* to bring up the *New SSO Configuration* dialogue. [Figure 2].
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/single-sign-on.adoc:190]:
```suggestion
```

**Review by AlexicaWright — COMMENTED** [review]
I've tested this and I have added some suggestions. In general, I think we should keep it pretty high level, our users are pretty savvy already. 
Additionally, we should probably add some information about role mapping? Link to that section in the docs?

**Comment by AlexicaWright** [comment]
org-settings

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-550.surge.sh

Updated pages:
- [Single Sign-On (SSO)](https://neo4j-docs-aura-550.surge.sh/aura/platform/security/single-sign-on)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-550.surge.sh

Updated pages:
- [Single Sign-On (SSO)](https://neo4j-docs-aura-550.surge.sh/aura/platform/security/single-sign-on)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-550.surge.sh

Updated pages:
- [Single Sign-On (SSO)](https://neo4j-docs-aura-550.surge.sh/aura/platform/security/single-sign-on)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-550.surge.sh

Updated pages:
- [Single Sign-On (SSO)](https://neo4j-docs-aura-550.surge.sh/aura/platform/security/single-sign-on)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-550.surge.sh

Updated pages:
- [Single Sign-On (SSO)](https://neo4j-docs-aura-550.surge.sh/aura/platform/security/single-sign-on)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-550.surge.sh

Updated pages:
- [Single Sign-On (SSO)](https://neo4j-docs-aura-550.surge.sh/aura/platform/security/single-sign-on)

**Comment by fiquick** [comment]
Hey Jessica, I've implemented the requested changes - thank you for the feedback!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-550.surge.sh

Updated pages:
- [Single Sign-On (SSO)](https://neo4j-docs-aura-550.surge.sh/aura/platform/security/single-sign-on)

**Review by AlexicaWright — APPROVED** [review]
LGTM!

**Comment by fiquick** [comment]
[SSO Set up instructions are incorrect in the current published Aura documentation](https://trello.com/c/nTgJPVDc/411-sso-set-up-instructions-are-incorrect-in-the-current-published-aura-documentation)

**Pull request merged**